server/controllers/refer/reasonController.ts

```diff
@@ -0,0 +1,60 @@
+import type { Request, Response, TypedRequestHandler } from 'express'
+import createError from 'http-errors'
+
+import { referPaths } from '../../paths'
+import type { PersonService, ReferralService } from '../../services'
+import { FormUtils, TypeUtils } from '../../utils'
+import type { ReferralUpdate } from '@accredited-programmes/models'
+
+export default class ReasonController {
+  constructor(
+    private readonly personService: PersonService,
+    private readonly referralService: ReferralService,
+  ) {}
+
+  show(): TypedRequestHandler<Request, Response> {
```

Comment:
Probably a good opportunity to start specifying the Params and Request Body types. Will look into it next week.

Reply:
That sounds good - maybe as a separate PR?